addons/project_sale_expense/models/project_project.py

```diff
@@ -64,14 +64,20 @@ def _get_expenses_profitability_items(self, with_action=True):
             total_amount_expense_to_invoice += currency._convert(revenues['to_invoice'], self.currency_id, self.company_id)
             total_amount_expense_invoiced += currency._convert(revenues['invoiced'], self.currency_id, self.company_id)
 
+        hr_expense_ids = self.env['hr.expense'].browse(expense_ids)
```

Comment:
```suggestion
        expenses = self.env['hr.expense'].browse(expense_ids)
```
:slightly_smiling_face: